helper-cli/src/main/kotlin/commands/packagecuration/CreateCommand.kt

```diff
@@ -56,27 +56,30 @@ internal class CreateCommand : CliktCommand(
     override fun run() {
         val outputFile = getSplitCurationFile(outputDir, packageId, FileFormat.YAML.fileExtension)
 
-        if (outputFile.exists()) {
-            throw UsageError("File ${outputFile.absolutePath} already exists.")
+        val curations = if (outputFile.isFile) {
+            outputFile.readValue<MutableSet<PackageCuration>>()
+        } else {
+            mutableSetOf()
         }
 
-        val mapper = createBlockYamlMapper()
+        if (packageId in curations.map { it.id }) {
+            println("Curation for ${packageId.toCoordinates()} already exists in '${outputFile.absolutePath}'.")
+        } else {
```

Comment:
Consider adding a return into the `if-`branch to eliminate one level of nesting.

Reply:
@fviernau Done.